apps/i18n/common/en_us.json

```diff
@@ -2013,6 +2013,8 @@
   "studentAnnouncementSpecial2020Heading": "Learn computer science at home",
   "studentAnnouncementSpecial2020Body": "Tune in to a CodeBytes mini-lesson, or see resources for students, parents, and teachers - including videos, fun tutorials, and projects!",
   "studentAnnouncementSpecial2020Button": "Get started",
+  "studentAnnouncementSpecial2022HocHeading": "Hour of Code: Explore, Play, Create",
+  "studentAnnouncementSpecial2022HocBody": "Explore the universe, score a goal, bust a move — there are so many ways to try computer science!",
```

Comment:
Props to the folks who will translate "bust a move" 😮

Reply:
We had a convo about this somewhere in Slack! 😅